app/_hub/kong-inc/openid-connect/how-to/_demonstrating_proof_of_possession.md

```diff
@@ -0,0 +1,103 @@
+This allows an OAuth2 upstream without DPoP support to work with the DPoP token without loosing the protection of key-binding mechanism.
+
+It's a prerequisite that the auth server is configured to enable the DPoP for you to enable it with Kong.
+
+If you are configuring this in Keycloak, see the [Keycloak configuration](#prerequisites) section in the prerequisites.
+For alternative auth servers, consult their documentation to configure this functionality.
+
+Some of the instructions in the other how-to guides for OpenID Connect support validation of access tokens using mTLS proof of possession.
+Enabling the [`proof_of_possession_dpop`](/hub/kong-inc/openid-connect/configuration/#config-proof_of_possession_dpop) configuration option in the plugin helps to ensure that the supplied access token
+belongs to the client by verifying its binding with the client certificate provided in the request.
+
+The DPoP are supported by the following auth methods:
+
+- [JWT Access Token Authentication](/hub/kong-inc/openid-connect/how-to/authentication/jwt-access-token/)
+- [Introspection Authentication](/hub/kong-inc/openid-connect/how-to/authentication/introspection/)
+- [Session Authentication](/hub/kong-inc/openid-connect/how-to/authentication/session/)
+
+   Session Authentication is only compatible with DPoP when used along with one of the other supported authentication methods:
+
+    * If multiple `openid-connect` plugins are configured with the `session` auth method, we strongly recommend configuring different values of [`session_secret`](/hub/kong-inc/openid-connect/configuration/#config-session_secret) across plugin instances for additional security. This avoids sessions being shared across plugins and possibly bypassing the proof of possession validation.
+
+The following example shows how to enable this feature for the JWT Access Token Authentication method. Similar steps can be followed for the other methods.
+
+## Demo
+### Prerequisites
+
+Follow these prerequisites to set up a demo Keycloak app and a Kong service and route for testing mTLS client auth.
+
+{% include_cached /md/plugins-hub/oidc-prereqs.md %}
+
+### Configure Demonstrating Proof-of-Possession
+
+1. To enable Demonstrating Proof-of-Possession, use the [`proof_of_possession_dpop`](/hub/kong-inc/openid-connect/configuration/#config-proof_of_possession_dpop) configuration option:
+
+    ```bash
+    http -f put :8001/plugins/5f35b796-ced6-4c00-9b2a-90eef745f4f9 \
+    name=openid-connect                                          \
+    service.name=openid-connect                                  \
+    config.issuer=https://keycloak.test:8440/auth/realms/master  \
+    config.client_id=cert-bound                                  \
+    config.client_secret=cf4c655a-0622-4ce6-a0de-d3353ef0b714    \
+    config.auth_methods=bearer                                   \
+    config.proof_of_possession_dpop=strict
+    ```
```

Comment:
Let's use this formatting:

https://github.com/Kong/docs.konghq.com/blob/main/app/_hub/kong-inc/openid-connect/how-to/client-authentication/_mtls.md?plain=1#L77-L96

That will generate a full example with all selected tools/formats: 
https://docs.konghq.com/hub/kong-inc/openid-connect/how-to/client-authentication/mtls/#set-up-mtls-for-the-oidc-plugin